### PR TITLE
Add example of inverted index for array

### PR DESCRIPTION
**Pull request by deeptikorwar**
Addresses #8216

### CODE REVIEW

**Review by rytaft — APPROVED** [review]
:lgtm: 

> Reviewed 2 of 2 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8630#-:-MJWq4Ui5BKbvRFVguYJ:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained

File: v20.1/inverted-indexes.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ If you require comparisons using [`<`](functions-and-operators.html#operator-les
 
 ## Example
 
+### Create table with inverted index on a JSONB column
```
Comment (Amruta-Ranade) [inline v20.1/inverted-indexes.md:147]: nit: "Create table" > "Create a table"

File: v20.1/inverted-indexes.md [hunk 2]
```diff
@@ -217,6 +220,72 @@ Now, run a query that filters on the `JSONB` column:
 (2 rows)
 ~~~
 
+### Add inverted index to a table with an array column
```
Comment (Amruta-Ranade) [inline v20.1/inverted-indexes.md:223]: nit: "Add inverted index" > "Add an inverted index"

File: v20.2/inverted-indexes.md [hunk 3]
```diff
@@ -144,6 +144,8 @@ If you require comparisons using [`<`](functions-and-operators.html#operator-les
 
 ## Example
 
+### Create table with inverted index on a JSONB column
```
Comment (Amruta-Ranade) [inline v20.2/inverted-indexes.md:147]: nit: "Create table" > "Create a table"

File: v20.2/inverted-indexes.md [hunk 4]
```diff
@@ -217,6 +220,72 @@ Now, run a query that filters on the `JSONB` column:
 (2 rows)
 ~~~
 
+### Add inverted index to a table with an array column
```
Comment (Amruta-Ranade) [inline v20.2/inverted-indexes.md:223]: nit: "Add inverted index" > "Add an inverted index"

**Pull request merged**